.ci/compute_projects_test.py

```diff
@@ -188,6 +188,19 @@ def test_exclude_gn(self):
         self.assertEqual(env_variables["runtimes_to_build"], "")
         self.assertEqual(env_variables["runtimes_check_targets"], "")
 
+     def test_ci(self):
```

Comment:
This function seems to be indented by five spaces not four which is why you're getting the formatting error.